recipes/kmc/3.1.1rc1/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "3.1.1" %}
+{% set rc = "rc1" %}
+{% set printed_version = "3.1.0" %}
+
+package:
+  name: kmc
+  version: {{ version }}{{ rc }}
+
+source:
+  url: https://github.com/refresh-bio/KMC/archive/v{{ version }}{{ rc }}.tar.gz
+  sha256: c7382df6af73f456fea2b497231958144fd01198280fa6427204de58434741f6
+  patches:
+    - 0001-fixes-for-conda.patch
+
+build:
+  number: 0
+  # this recipe builds kmc without the asmlib speedup
+
+requirements:
+  build:
+    # C++14 features are required
```

Comment:
Are you sure about this comment? You are not using the new compilers as it seems.

Reply:
@bgruening the makefile does specify C++14: https://github.com/bioconda/bioconda-recipes/pull/9952/files#diff-10ea6eb45d548abe3d44bba556156971R32